Macros/ACS_summary_geo_source_bg_vars.sas

```diff
@@ -858,20 +993,8 @@
       mNumRentCstBurden_50_74K_&_years. = "Renter-occupied housing units with housing cost burden and household income $50,000 to $74,999, MOE, &_years_dash  "
       mNumRentCstBurden_75_99K_&_years. = "Renter-occupied housing units with housing cost burden and household income $75,000 to $99,999, MOE, &_years_dash  "
       mNumRentCstBurden_GT100K_&_years. = "Renter-occupied housing units with housing cost burden and household income $100,000 or more, MOE, &_years_dash  "
-    mGrossRentLT100_499_&_years. = "Renter-occupied housing units where gross rent is less than $100 to $499, MOE, &_years_dash "
-      mGrossRent500_799_&_years. = "Renter-occupied housing units where gross rent is $500 to $799, MOE, &_years_dash "
-      mGrossRent800_899_&_years. = "Renter-occupied housing units where gross rent is $800 to $899, MOE, &_years_dash "
-      mGrossRent900_999_&_years. = "Renter-occupied housing units where gross rent is $900 to $999, MOE, &_years_dash "
-      mGrossRent1000_1249_&_years. = "Renter-occupied housing units where gross rent is $1000 to $1249, MOE, &_years_dash "
-      mGrossRent1250_1499_&_years. = "Renter-occupied housing units where gross rent is $1250 to $1499, MOE, &_years_dash "
-      mGrossRent1500_1999_&_years. = "Renter-occupied housing units where gross rent is $1500 to $1999, MOE, &_years_dash "
-      mGrossRent2000_2499_&_years. = "Renter-occupied housing units where gross rent is $2000 to $2499, MOE, &_years_dash "
-      mGrossRent2500_2999_&_years. = "Renter-occupied housing units where gross rent is $2500 to $2999, MOE, &_years_dash "
-      mGrossRent3000_3499_&_years. = "Renter-occupied housing units where gross rent is $3000 to $3499, MOE, &_years_dash "
-      mGrossRentGT3500_&_years. = "Renter-occupied housing units where gross rent is greater than $3500, MOE, &_years_dash "
   medianhomevalue_&_years. = "Median value of owner-occupied housing units ($),&_years_dash"
   mmedianhomevalue_&_years.="Median value of owner-occupied housing units ($), MOE, &_years_dash"
-
```

Comment:
There was an error that seemed to be generated by this label statement, but I couldn't see what the cause was. Please check again when you re-run. 

```
ERROR 22-322: Syntax error, expecting one of the following: a name, a quoted string, a numeric constant, 
              a datetime constant, a missing value, INPUT, PUT.  
```